js/settingsview.js

```diff
@@ -36,7 +36,7 @@
 		+ '     <span><small>' + t('twofactor_u2f', 'You can add as many devices as you like. It is recommended to give each device a distinct name.') + '</small></span>'
 		+ '	{{else}}'
 		+ '     <span class="icon-loading-small u2f-loading"></span>'
-		+ '	<span>' + t('twofactor_u2f', 'Adding a new device …') + '</span>'
+		+ '	<span>' + t('twofactor_u2f', 'Adding a new device…') + '</span>'
```

Comment:
Whatever the paradigm, it must be consistent across the whole project. Right now it isn't.
Not that it is a pressing issue, given there are more things to handle, but I think 
*… makes for fewer errors in translation.

Sticking to one way, and not invoking or otherwise require thought on part of the translators would be ideal.

To elaborate on the topic, this is one of the strings:
> Smile in 3… 2… 1!

One could about that, "what is that…"  or by writing  
"Then there is this "Smile in 3… 2… 1!" …" To maybe signify the confusion there being a thing onto itself.
I think the strict use there would be to say * […]

However, luckily there isn't any of that going on in Nextcloud.

In Nextcloud, ellipsis is invariably used to note 
an action/verb in infinitive tense "Hide description…"
preteritum-  "Verifying…"
or 
as an omission "Server address https://…"

A common use, described in some style-guides, and employed in some languages, is the use of ellipsis to replace words. This becomes an issue with things like "Announcing" (which is an actual string) followed by *… means the action, or in the case of * … "Announcing …" would signify a subjective being announced.

The problem then becomes:
"This i can not see…" Where it could be interpreted as unseen, or possibly not see-sawable.
(A see-saw being a type of saw.) Which isn't a problem if you aren't extremely unlucky and unaware.
This is possibly why in the strictest sense, you would do …. ellipsis with trailing dot, to mark an end an end of a sentence where the intention was to first append, and then end.

As long as you avoid words that could feasibly be added onto by the use of an ellipsis, *… aligns better with interpretation and the use in Nextcloud.